app/models/parent_object.rb

```diff
@@ -127,7 +127,7 @@ def ladybird_json=(lb_record)
     return lb_record if lb_record.blank?
     self.last_ladybird_update = DateTime.current
     return unless use_ladybird
-    self.bib = lb_record["orbisBibId"] || lb_record["orbisRecord"]
+    self.bib = lb_record["orbisBibId"]
```

Comment:
If we have to change all our fixture objects to accommodate this change, does that maybe indicate we should leave the "or" statements for now, until we've re-fetched everything from the new version of the metadata cloud in the production-like environments?

Then we can re-fetch all our fixtures from the metadata cloud, make sure everything is still passing, and do the updates to the fixtures in a separate PR?